src/openzeppelin/access/accesscontrol.cairo

```diff
@@ -0,0 +1,148 @@
+use starknet::ContractAddress;
+
+#[abi]
+trait IAccessControl {
+    fn has_role(role: felt252, account: ContractAddress) -> bool;
+    fn get_role_admin(role: felt252) -> felt252;
+    fn grant_role(role: felt252, account: ContractAddress);
+    fn revoke_role(role: felt252, account: ContractAddress);
+    fn renounce_role(role: felt252, account: ContractAddress);
+}
+
+#[contract]
+mod AccessControl {
+    // OZ modules
+    use super::IAccessControl;
+    use openzeppelin::introspection::erc165::ERC165;
+
+    // Constants
+    use openzeppelin::utils::constants::DEFAULT_ADMIN_ROLE;
```

Comment:
I mean, we are just using both constants once, so we can just import the constants mod and do `constants::<target>`. I went back and forth on this originally, but thinking about it now, it might be better that way

Reply:
i think either way is fine, although we should settle on this since changing it later might be a breaking change. @ericnordelo do you have any preferences between having `ACCOUNT_ID` and `DEFAULT_ADMIN_ROLE` as constants in the `constants` module vs in their respective modules e.g. `openzeppelin::account::ACCOUNT_ID` and `openzeppelin::access::access_control::DEFAULT_ADMIN_ROLE`? i think i lean towards the latter

Reply:
For me also both ways are fine as long as we keep consistency, but I'm leaning towards the latter as well (respective modules), mostly because I think you get more info regarding these constants this way just by looking at the imports.